lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -412,7 +412,7 @@ <h4 class="message-title">${_('Your current verification will expire soon.')}</h
               </p>
               <div class="action-upgrade-container">
                 % if use_ecommerce_payment_flow and course_mode_info['verified_sku']:
-                  <a id="ecommerce_upgrade_link" class="action action-upgrade" href="${ecommerce_payment_page}?sku=${course_mode_info['verified_sku']}">
```

Comment:
looks good assuming this id is one that we added, i wouldn't remove if the id was already there before this ticket

Reply:
Yup, we originally added the ID at https://github.com/edx/edx-platform/pull/24338/files#diff-d4bf11344f0cb0d0fb76cd819987a5feR415

Reply:
And the user can only click on one upgrade button at a time, so there is also no need to know which upgrade button was clicked I'm assuming

Reply:
Yup!